workflow/README.md

```diff
@@ -5,14 +5,19 @@ Automate your data processing tasks using Peak Workflows.
 We provide Python and R versions of the images:
 - [Python](./python)
   - [standard-pack](./python/standard-pack)
-    - [workflow-python-standard-pack-v1](./python/standard-pack/v1)
+    - [workflow-python-standard-pack-1.0.0-base-python-3.8.13](./python/standard-pack/1.0.0-base-python-3.8.13)
   - [ds-pack](./python/ds-pack)
-    - [workflow-python-ds-pack-v1](./python/ds-pack/v1/)
+    - [workflow-python-ds-pack-1.0.0-base-python-3.8.13](./python/ds-pack/1.0.0-base-python-3.8.13/)
+    - [workflow-python-ds-pack-1.1.0-base-python-3.8.18](./python/ds-pack/1.1.0-base-python-3.8.18/)
```

Comment:
Hey @peak-jiteshahuja, with workspaces, we upgraded version to `2.0.0`. Should we do the similar here

Reply:
Discussed offline. We might not need this.